.travis.yml

```diff
@@ -55,4 +54,4 @@ script:
   - bundle exec rails db:structure:load
   - bundle exec rails db:seed
   - rake ci
-  - bundle exec codeclimate-test-reporter
+  - CODECLIMATE_REPO_TOKEN=861a17b4c76ddedf9972cdaca2771f9ab0c99f4ce92298fa965d25a4be55bdbe bundle exec codeclimate-test-reporter
```

Comment:
Generally it's not, it's not strange, it's his feature, other servers do not do that, I ran into this when I connected it to the repository on node.js

https://github.com/bbatsov/rubocop/blob/master/.travis.yml

That's why this token is used when the command is not in the environment variable